### PR TITLE
feature: All events page

### LINKED ISSUE #1882: Events page
Create an events page and topbar link.

On the events page we want to show all events in chronological order, and a past events tab showing events in reverse-chronological order

### PR DESCRIPTION
**Pull request by darrenvong**
Closes #1882.

Currently blocked by the bug in #1945 when you click onto the past events tab.

**Desktop**
Screenshot 2021-09-18 at 04 29 26 |

**Mobile**
Screenshot 2021-09-18 at 04 29 09

**Web frontend checklist**
- [x] Formatted my code with `yarn format && yarn lint --fix`
- [x] There are no warnings from `yarn lint`
- [x] There are no console warnings when running the app
- [ ] Added any new components to storybook
- [x] Added tests where relevant
- [x] All tests pass
- [x] Clicked around my changes running locally and it works
- [x] Checked Desktop, Mobile and Tablet screen sizes

### CODE REVIEW

File: app/web/src/features/communities/events/EventsTab.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+  return (
+    <div className={classes.root}>
+      {!pastEvents && (
+        <Button onClick={() => history.push(newEventRoute)}>
+          {CREATE_AN_EVENT}
+        </Button>
+      )}
+      <Typography variant="h2">{tabTitle}</Typography>
+      {error && <Alert severity="error">{error.message}</Alert>}
```
Comment (lucaslcode) [inline app/web/src/features/communities/events/EventsTab.tsx]: IMO it makes more sense to have title - error - create button

**Review by lucaslcode — COMMENTED** [review]
Looks pretty good

**Comment by lucaslcode** [comment]
#1945 is fixed

File: app/web/src/features/communities/events/EventsTab.tsx (hunk 1 above)
Comment (darrenvong) [reply app/web/src/features/communities/events/EventsTab.tsx]: Hmm yeah that'd also be more consistent with the other pages 👍

**Review by lucaslcode — APPROVED** [review]

**Pull request merged**